cloud_registry/version

```diff
@@ -0,0 +1 @@
+0.2.0
```

Comment:
had to create a separate version text file because pyproject.toml config does not support reading version from a python variable.
we will have to either define the version in `pyproject.toml` itself or define it in a separate file as above

Reply:
I might not be aware. Need pointers to why are we using `pyproject.toml` instead of setup config? Is there any deprecation coming around or already published?

Reply:
yes, as per PEP 621 standard, python community has decided to use pyproject.toml as the main source of configuration for the build process.
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

Reply:
If you can define it in `pyproject.toml` itself, then great. If we need it elsewhere, it shouldn't be too hard to parse it from there (at least not compared to a text file)